MODULE.bazel

```diff
@@ -58,6 +58,7 @@ register_toolchains("@pythons_hub//:all")
 
 pip = use_extension("//python/extensions:pip.bzl", "pip")
 pip.parse(
+    experimental_index_url = "https://pypi.org/simple",
```

Comment:
Is it intentional not to use PIP_INDEX_URL env here, and why?

Reply:
If you want to override this value you can use bazel downloader config. In the general sense it is not a given that people will have their `PIP_INDEX_URL` configured to contain the index that works for fetching these dependencies.

Hence to avoid unintended breakage, this is hardcoded.